lib/bolt/pal.rb

```diff
@@ -476,6 +476,29 @@ def list_plans(filter_content: false)
       end
     end
 
+    def list_scripts
+      in_bolt_compiler do
+        scripts = []
+
+        Puppet.lookup(:current_environment).modules.each do |mod|
+          if mod.files?
+            if File.directory?(file_scripts = File.join(mod.files, 'scripts'))
+              Dir.children(file_scripts).each do |script|
```

Comment:
Does script discovery need to descend into nested directories (and *not* list directory names themselves?).

Reply:
Ah, good catch, it does.

Reply:
Updated to recursively search for scripts. 👍